cmd/run.php

```diff
@@ -37,7 +38,30 @@
     GithubScrapper::TWITTER => $scrapper->getTwitter(),
     GithubScrapper::EMAIL => $scrapper->getEmail(),
     GithubScrapper::VERIFICATION => $scrapper->getVerification(),
-    default => "unknown option"
+    default => $climate->error("unknown option"),
 };
 
 $climate->out($result);
+
+$breweryName = $climate->input("Please provide brewery or city name:")->prompt();
+$breweryScrapper = new BreweryScrapper($breweryName, new Client());
+$climate->br();
+$breweryScrapper->dataToArray();
+$breweryScrapper->getShowLabel($breweryName);
+
+while (true) {
+    $climate->br();
+    $breweryOption = $climate->radio("Choose data to fetch:", [
+        BreweryScrapper::LIST => "showing a list of providers",
+        BreweryScrapper::CLEAR => "clearing cache",
+        BreweryScrapper::EXIT => "exit",
+    ])->prompt();
+
+    $beweryResult = match ($breweryOption) {
+        BreweryScrapper::LIST => $breweryScrapper->getList(),
+        BreweryScrapper::CLEAR => $breweryScrapper->getClear(),
+        EXIT => $climate->exit(),
```

Comment:
> `scrapperBrewery` should not be responsible to exit console. Use climate instead.


It was a suggestion from @mtracz

Reply:
True, that was my suggestion
Previous
```php
BreweryScrapper::EXIT => $scrapperBrewery->getExit(),
```
now:
```php
EXIT => $climate->exit(),
```
So that is good, you remove responsibility for a exit from the console, but const name must stay here or think how to refactor it.

`BreweryScrapper::EXIT` is a const name `public const EXIT = "exit";`.

Reply:
we changed it.